src/curl.c

```diff
@@ -360,6 +362,13 @@ enum download_status process_curl_error_codes(int curl_ret, CURL *curl_handle)
 		url = "<not available>";
 	}
 
+	/*
+	 * retrieve bytes transferred, errors or not
+	 */
+	curl_off_t curl_sz;
+	curl_easy_getinfo(curl_handle, CURLINFO_SIZE_DOWNLOAD_T, &curl_sz);
```

Comment:
Culr documentation say we can't trust the result value if this function returs anything other than CURLE_OK. I'm ok on just checking that and ignoring the result if it's not CURLE_OK.